internal/mock/store/store.go

```diff
@@ -0,0 +1,210 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: internal/store/store.go
+
+// Package mock_store is a generated GoMock package.
+package mock_store
```

Comment:
Hi @sgotti. I added `go:generate` line like below in `store.go` file
```go
//go:generate mockgen -destination=../mock/store/store.go -source=$GOFILE
```
and when we do `go generate ./...` from our project directory, it is generating the file at the given destination. 

I am facing one **problem** there. While generating it is using local import for store like
```go
import (
	x "."
	context "context"
)
```
instead of 
```go
import (
	store "github.com/sorintlab/stolon/internal/store"
	context "context"
```

I spent some time to resolve this. But no luck. Please help me to fix this. In the mean time, I will also try to fix this. Thanks